tla-import/src/test/scala/at/forsyte/apalache/tla/imp/TestSanyImporter.scala

```diff
@@ -1957,45 +1962,52 @@ class TestSanyImporter extends FunSuite {
     assert(13 == root.declarations.size)
   }
 
-
-  /*
-  TODO: we need a good way to propagate this module to the standard library
-
-  test("module BMC") {
-    // check that the module BMC is imported properly
+  test("EXTENDS Apalache") {
     val text =
-      """---- MODULE bmc ----
-        |EXTENDS BMC
+      """---- MODULE root ----
+        |EXTENDS Integers, FiniteSets, Apalache
+        |VARIABLE x, S
         |
-        |VARIABLES i
-        |
-        |AWithType == WithType(i, "IntT")
+        |Assn == x' := 1
+        |Sklm == Skolem(\E y \in S: TRUE)
+        |Expnd == Expand(SUBSET S)
+        |CC == ConstCardinality(Cardinality(S) >= 2)
         |================================
-        |""".stripMargin
+      """.stripMargin
 
-    val (rootName, modules) = new SanyImporter().loadFromSource("bmc", Source.fromString(text))
-    assert(2 == modules.size) // our module
-    // the root module and naturals
-    val root = modules(rootName)
+    // We have to set TLA-Library, in order to look up for Apalache.tla. This is done automatically in pom.xml.
+    // If you run this test in an IDE, and the test fails, add the following line to the VM parameters
+    // (don't forget to replace <APALACHE_HOME> with the directory where you checked out the project):
+    //
+    // -DTLA-Library=<APALACHE_HOME>/src/tla
+    System.out.println("TLA-Library = %s".format(System.getProperty("TLA-Library")))
 
-    def assertTlaDecl(expectedName: String, body: TlaEx): Unit = {
-      root.declarations.find {
-        _.name == expectedName
-      } match {
-        case Some(d: TlaOperDecl) =>
-          assert(expectedName == d.name)
-          assert(0 == d.formalParams.length)
-          assert(body == d.body)
+    val locationStore = new SourceStore
+    val (rootName, modules) = new SanyImporter(locationStore)
+      .loadFromSource("root", Source.fromString(text))
+    assert(6 == modules.size) // root, Apalache, Integers, FiniteSets, and whatever they import
```

Comment:
This test seems a bit brittle to me. iiuc, it requires there to be 6 modules to be imported (including transitive dependencies?). Do we actually want this to fail any time one of these modules adds a new dependency? If not, perhaps we should check for the presence of the modules that we know should be included?

Reply:
I do see this is how we do it through the tests here, so probably better to leave just for consistency.

Reply:
Good point. However, the standard TLA+ modules rarely change. So it may be good if the tests fail with a new version of `tla2tools`. This will at least indicate a structural change.